compiler/ras/Debug.cpp

```diff
@@ -1967,6 +1967,17 @@ TR_Debug::getStaticName(TR::SymbolReference * symRef)
       if (sym->isConst())
          return "<constant>";
 
+      // Value Type default value instance slot address
+      if (sym->isStatic() && staticAddress)
```

Comment:
Sorry for the dumb question.  Are there any static symbols that are not default values that might get caught by this condition?  Symbols that would previously have fallen through to the return at the end of the method?  I'm just wondering why it's sufficient to check `isStatic` rather than `isStaticDefaultValueInstance`. . . .

Reply:
Good point! Although I'm not sure of other cases fall into this path, but I think here I should check `isStaticDefaultValueInstance` instead.

Reply:
Addressed in 1cd0e50